test/functionalTest/cases/1552_invalid_sub_id/invalid_sub_id.test

```diff
@@ -52,28 +52,28 @@ echo
 01. Ask to see a subscription with invalid ID
 =============================================
 HTTP/1.1 400 Bad Request
-Content-Length: 69
+Content-Length: 94
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "description": "bad length - 24 chars expected",
+    "description": "Invalid subscription ID: bad length - 24 chars expected",
     "error": "BadRequest"
 }
 
 
 02. Ask to see a subscription that doesn't exist
 ================================================
-HTTP/1.1 400 Bad Request
-Content-Length: 89
+HTTP/1.1 404 Not Found
+Content-Length: 97
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "description": "",
-    "error": "subscriptionId does not correspond to an active subscription"
+    "description": "subscriptionId does not correspond to an active subscription",
+    "error": "NotFound"
```

Comment:
This change in .test (and some other similar) corresponds to the following entry in CNR: "Fix: GET /v2/subscriptions/subId error for non existing subscriptions (now using 404 Not Found, previously using 400 Bad Request)".

NTC (just a remark)